implementations/rust/ockam/ockam_key_exchange_x3dh/src/lib.rs

```diff
@@ -53,7 +55,7 @@ impl core::fmt::Debug for Signature {
 }
 
 /// Represents all the keys and signature to send to an enrollee
-#[derive(Clone, Debug)]
+#[derive(Clone, Debug, Zeroize)]
```

Comment:
Do we not want `#[zeroize(drop)]` here too?

Reply:
Oops. Added a commit and squashed...